bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooworkspace.py

```diff
@@ -11,6 +11,29 @@
 
 from ._utils import _kwargs_to_roocmdargs, cpp_signature
 
+def make_json_for_variable(var_name, var_dict):
+    val, max_value, min_value, is_constant = [None, None, None, True]
+    if "value" in var_dict:
+        val = var_dict["value"]
+    if ("max" in var_dict) and ("min" in var_dict):
+        max_value, min_value, is_constant = var_dict["max"], var_dict["min"], False
+    if not (val is None) or not ((max_value is None) and (min_value is None)):
+        if(val is None):
+            val = (max_value+min_value)/2
+        # Create dictionary with domains and parameter points
+        json_dict = {
+            "domains": [ { "axes": [ {"name": var_name} ], "name": "default_domain", "type": "product_domain" } ],
+            "parameter_points": [ { "name": "default_values", "parameters": [ { "name": var_name, "value": val } ] } ] 
+        }
+        if is_constant:
+            json_dict["parameter_points"][0]["parameters"][0]["const"] = True
+            json_dict["misc"] = { "ROOT_internal": { var_name: { "tags" : "Constant" } } }
+        if not ((max_value is None) and (min_value is None)):
+            json_dict["domains"][0]["axes"][0]["max"] = max_value
+            json_dict["domains"][0]["axes"][0]["min"] = min_value
+        return json_dict
+    else:
+        raise SyntaxError("Invalid Syntax: Please provide either 'value' or 'min' and 'max' or both")
```

Comment:
```suggestion
        raise ValueError("Invalid Syntax: Please provide either 'value' or 'min' and 'max' or both")
```
I would rather go for `ValueError` here. `SyntaxError` is for errors in the actual Python syntax. Yes, I see why you used it because it's about our JSON Syntax, but from the perspective of Python the dictionaries are just values.